sys/net/gnrc/network_layer/ndp/gnrc_ndp.c

```diff
@@ -632,6 +712,20 @@ void gnrc_ndp_rtr_adv_handle(kernel_pid_t iface, gnrc_pktsnip_t *pkt, ipv6_hdr_t
         }
     }
 #endif
+
+#if defined(MODULE_GNRC_RPL) && defined(MODULE_GNRC_SIXLOWPAN_ND_BORDER_ROUTER)
+    if (!(if_entry->flags & GNRC_IPV6_NETIF_FLAGS_SIXLOWPAN)) {
+        ipv6_addr_t default_route_prefix;
```

Comment:
you can initialize this statically by using `ipv6_addr_t default_route_prefix = IPV6_ADDR_UNSPECIFIED;` and ignore the `memset` below